CHANGELOG.md

```diff
@@ -50,22 +50,22 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [0.8c beta] - 2018-10-21
 ### Added
-- messge box promping for a reboot when changing update facilitator settings
+- message box prompting for a reboot when changing update facilitator settings
 - tooltips to list view for long texts
 
 ### Changed
 - some buttons are now disabled when no updates are checked
 
 ### Fixed
-- issue with supprot url's nor manualy generated mased on the kb number
+- issue with support url's not manually generated based on the kb number

```

Comment:
This should be `URLs` and `manually-generated`.

Reply:
"Manually-generated" is a compounded adverb; it's effectively saying "the generation (verb) was done manually (adverb)", but as a single (hyphenated) word.  I will admit this one can go either way (hyphenated or not), so I will bend on it.  English is a terrible language in many regards, and this is one of them.

But more importantly: this is a GitHub PR, not an English lesson.